packages/react-conformance/src/types.ts

```diff
@@ -102,12 +90,12 @@ export interface IsConformantOptions<TProps = {}> {
   tsconfigDir?: string;
 }
 
-export type ConformanceTest<TProps = {}> = (
+export type ConformanceTest<TProps = {}, TExtraOptions = {}> = (
```

Comment:
why is this change of API needed ?

Reply:
I think left over from when I was trying to implement northstar's `as` prop tests with the react-conformance infrastructure? But it may no longer be needed since I ended up restoring northstar's `as` prop tests to the way they were implemented prior to react-conformance.

Reply:
sounds good, so can be removed/undo right ?